tests/cli/test_main_notices.py

```diff
@@ -214,3 +231,66 @@ def test_cache_names_appear_as_expected(
 
         assert len(cache_files) == 1
         assert os.path.basename(cache_files[0]) == expected_cache_filename
+
+
+def test_notices_appear_once_when_running_decorated_commands(
+    tmpdir, env_one, notices_cache_dir, pre_link_messages_package
+):
+    """
+    As a user, I want to make sure when I run commands like "install" and "update"
+    that the channels are only appearing according to the specified interval in:
+        conda.base.constants.NOTICES_DECORATOR_DISPLAY_INTERVAL
+
+    This should only be once per 24 hours according to the current setting.
+
+    To ensure this test runs appropriately, we rely on using a pass-thru mock
+    of the `conda.notices.fetch.get_notice_responses` function. If this function
+    was called and called correctly we can assume everything is working well.
+    """
+    offset_cache_file_mtime(NOTICES_DECORATOR_DISPLAY_INTERVAL + 100)
+
+    with mock.patch(
+        "conda.notices.fetch.get_notice_responses", wraps=fetch.get_notice_responses
+    ) as fetch_mock:
+        # First run of install; notices should be retrieved
+        run(
+            f"conda install -n {env_one} -c local --override-channels -y pre_link_messages_package"
+        )
+
+        # make sure our fetch function was called correctly
+        fetch_mock.assert_called_once()
+        args, kwargs = fetch_mock.call_args
+        # "wraps" probably makes this more nested than it normally would be, but the docs
+        # were not clear on that.
+        ((url, name), *_), *_ = args
+
+        assert url.startswith("file://")
+        assert name == "local"
+
+    with mock.patch(
+        "conda.notices.fetch.get_notice_responses", wraps=fetch.get_notice_responses
+    ) as fetch_mock:
```

Comment:
We can combine this with the prior context and just reset the mock wrapper:

```suggestion
        fetch_mock.reset_mock()
```